src/git/github-git/repo-commit.ts

```diff
@@ -0,0 +1,30 @@
+import { getOctokit } from "@actions/github";
+import {
+  CreateCommitOnBranchInput,
+  CreateCommitOnBranchPayload,
+} from "../../generated/graphql";
+
+export async function createCommitOnBranch(
```

Comment:
Only reference to this is in `index.ts#commitAll` seems like it can live in the same file.